src/main/java/com/simibubi/create/content/contraptions/relays/elementary/IEncased.java

```diff
@@ -0,0 +1,11 @@
+package com.simibubi.create.content.contraptions.relays.elementary;
+
+import net.minecraft.world.level.block.Block;
+
+/**
+ * Implement this interface to indicate that this block is an EncasedBlock
+ */
+public interface IEncased {
```

Comment:
The I- prefix is legacy and should not be used on new interfaces. This should be renamed to `Encased`.